infrastructure/collections/src/main/java/tech/pegasys/teku/infrastructure/collections/LimitStrategy.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.infrastructure.collections;
+
+public enum LimitStrategy {
+  DROP_LEAST_RECENTLY_ACCESSED,
+  DROP_OLDEST_ELEMENT
+}
```

Comment:
Do we actually need these two strategies?

Reply:
At the moment I don't think so. Will remove this class.